_dashboards/index.md

```diff
@@ -11,15 +11,57 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# OpenSearch Dashboards
+# Overview of OpenSearch Dashboards
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is the preferred visualization platform for analyzing data in OpenSearch. and is the user interface for many OpenSearch features. From real-time application monitoring, threat detection, and incident management to personalized search experience, OpenSearch Dashboards gives you data visualization tools to tell your data story.      
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and the OpenSearch suite provides [plugins](https://opensearch.org/docs/latest/dashboards/install/plugins/) to extend and customize dashboard features and functions. 
 
-## Get started with OpenSearch Dashboards
+<img src="{{site.url}}{{site.baseurl}}/images/get-started-dashboards.png)" alt="OpenSearch Dashboards visualizations"/>
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+# OpenSearch Dashboards use cases
+
+OpenSearch Dashboards makes it easy makes it easy for you to perform interactive log analytics, real-time application monitoring, website search, and more.  
+
+<table>
+<tbody>
+<tr>
+<td style="text-align: center; vertical-align: top;">Visualize log and trace data with interactive log analytics<img src="{{site.url}}{{site.baseurl}}/images/trace_log_correlation.gif)" alt="Log analytics"/></td>
+<td style="text-align: center; vertical-align: top;">Detect and mitigate issues faster with anomaly detection<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection"/></td>
+</tr>
+<tr>
+<td style="text-align: center; vertical-align: top;">Use observability to monitor and issue alerts<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Data overview" /></td>
```

Comment:
If we're following the "task/feature" pattern of the first two items, it would be:

```suggestion
<td style="text-align: center; vertical-align: top;">Monitor and issue alerts with observability<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Data overview" /></td>
```